firmwares/inovelli/LZW31-SN.json

```
@@ -25,6 +25,23 @@
 					"integrity": "sha256:e07cb9972bfe88e143c72c9b6ed88a640b9d969fbf09d6c1a4625f711979b541"
 				}
 			]
+		},
+		{
+			"version": "1.57",
+			"channel": "stable",
+			"changelog": "Fixed - Optimization of Aux mode in neutral setting. – Fixed - The inoperative issue after changing parameter 8 from high value to low value.",
```

```suggestion
			"changelog": "Fixed - Optimization of Aux mode in neutral setting.\nFixed - The inoperative issue after changing parameter 8 from high value to low value.",
```
Not sure if HA's renders this as markdown currently, but we should at least attempt to have a line break here.